lists/python/linked_list.py

```diff
@@ -24,11 +24,11 @@ def __eq__(self, other):
 
 
 class LinkedList(List):
+     """
```

Comment:
This won't cause any errors but the start of this comment has an extra space that throws off the alignment. Would you mind fixing those? It looks like it has happened in several places.